NOTICE.txt

```diff
@@ -31,3 +31,57 @@ bring it to our attention. Post an issue or email us:
 The attached notices are provided for information only.
 
 ------------------------------------------------------------------------------
+Copyright (c) 2012 - 2017, Lars Bilke
+Copyright (c) 2007      Daniel Gollub <dgollub@suse.de>
+Copyright (c) 2007-2018 Andreas Schneider <asn@cryptomilk.org>
+Copyright (c) 2018      Anderson Toshiyuki Sasaki <ansasaki@redhat.com>
```

Comment:
The license doesn't contain copyright info.
https://github.com/xbmc/libssh/blob/667fb5f9a9c96f210583dbfb11755c43250c5e55/cmake/Modules/COPYING-CMAKE-SCRIPTS#L1

```suggestion
```

Reply:
it is the file the one that mentions the names, that's why we need to merge both